### ISSUE TITLE
Support for original filename in exportFile

### ISSUE THREAD

**Issue by 123andy** [IUREDCap/phpcap#8]
I glanced through the code but do not think it supports a method to pull the original uploaded file information which is hidden at the end of the original REDCap docs:

> How to obtain the filename of the file:
> The MIME type of the file, along with the name of the file and its extension, can be found in the header of the returned response. Thus in order to determine these attributes of the file being exported, you will need to parse the response header. Example: `content-type = application/vnd.openxmlformats-officedocument.wordprocessingml.document; name='FILE_NAME.docx'`

Would be great to have an option to obtain that information from the file details during export, maybe exportFileObject which would return an object like:
```
{
  "content-type": "application/vnd.openxmlformats-officedocument.wordprocessingml.document",
  "name": "FILE_NAME.docx',
  "source": "...."
}
```

Or, I could be missing it if is already exists...  Thx.

**Comment by mullen2** [comment]
Hi Andy,

There is still only the method that I posted before here: https://community.projectredcap.org/idea/23449/an-all-new-shiny-phpcap-we-want-your-input.html. I have been focusing my REDCap effort on REDCap-ETL for a while, but I hope to be working on PHPCap again soon. Thanks for posting this as a GitHub issue.

Jim

**Comment by mullen2** [comment]
Andy,

Thanks for your persistence on this issue. I should have added it as a GitHub issue when you first brought it up. I just released a new version (1.2.1) that adds a fileInfo parameter to the exportFile method, which will be set to the information you mention. Your proposed solution may be better from a usability perspective than what I implemented, but I didn't want to add a largely redundant method or break backward compatibility, so I went with the additional parameter approach. Below is an example program.

Jim

```php
<?php 
use IU\PHPCap\RedCapProject;

require(__DIR__ . '/../../vendor/autoload.php');

$config = parse_ini_file(__DIR__.'/./config.ini');

$project = new RedCapProject(
    $config['api.url'],
    $config['api.token']
);
    

$fileInfo = array();
$fileContents = $project->exportFile(
    $recordId = '1001',
    $field = 'patient_document',
    $event = 'enrollment_arm_1',
    $instance = null,
    $fileInfo
);

print_r($fileInfo);

file_put_contents($fileInfo['name'], $fileContents);
```
Output:
```
Array 
(
    [mime_type] => text/plain
    [name] => file.txt
    [charset] => UTF-8
)
```

**Comment by 123andy** [comment]
Wow - I'm impressed.  I can't find time to tie my shoelaces.
